packages/edit-site/src/components/page-templates/dataviews-templates.js

```diff
@@ -31,17 +36,28 @@ import {
 	deleteTemplateAction,
 	renameTemplateAction,
 } from './template-actions';
+import usePatternSettings from '../page-patterns/use-pattern-settings';
+import { unlock } from '../../lock-unlock';
+
+const { ExperimentalBlockEditorProvider } = unlock( blockEditorPrivateApis );
 
 const EMPTY_ARRAY = [];
 
+const defaultConfigPerViewType = {
+	list: {},
+	grid: {
+		mediaField: 'preview',
+	},
+};
+
 const DEFAULT_VIEW = {
 	type: 'list',
 	search: '',
 	page: 1,
 	perPage: 20,
 	// All fields are visible by default, so it's
 	// better to keep track of the hidden ones.
-	hiddenFields: [],
+	hiddenFields: [ 'preview' ],
```

Comment:
I still consider a bug that the preview field cannot be hidden in the grid layout :/ Not to fix in this PR, though.

https://github.com/WordPress/gutenberg/assets/583546/9d9b797c-9e9e-4f3c-8e73-fc8ac6c573bc